### PR TITLE
Documented what's new in accessibility for .NET Framework 4.8

### PR DESCRIPTION
**Pull request by rpetrusha**
## Documented what's new in accessibility for .NET Framework 4.8

//cc @jimcarley @M-Lipin @miguep @NKarnam15 @vivmishra

### CODE REVIEW

**Review by jimcarley — APPROVED** [review]

File: docs/framework/whats-new/whats-new-in-accessibility.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "What's new in accessibility in the .NET Framework"
 ms.custom: "updateeachrelease"
-ms.date: "04/10/2018"
+ms.date: "04/18/2018"
```
Comment (mairaw) [inline docs/framework/whats-new/whats-new-in-accessibility.md]:
```suggestion
ms.date: "04/18/2019"
```

File: docs/framework/whats-new/whats-new-in-accessibility.md [hunk 2]
```diff
@@ -55,9 +65,260 @@ Applications that target versions of the .NET Framework starting with 4.7.1 can
...
+
+<a name="winforms48" />
+
+### Windows Forms
+
+In .NET Framework 4.8, Windows Forms adds support for LiveRegions and Notification Events to many commonly used controls. It also adds support for ToolTips when a user navigates to a control by using the keyboard.
+
+**UIA LiveRegions Support in Labels and StatusStrips**
+
+UIA LiveRegions allow application developers to notify screen readers of a text change in a control that is located apart from the location where the user is working. This is useful, for example,for a <xref:System.Windows.Forms.StatusStrip> control that shows a connection status. If the connection is dropped and the status changes, the developer might want to notify the screen reader.
```
Comment (mairaw) [inline docs/framework/whats-new/whats-new-in-accessibility.md]:
```suggestion
UIA LiveRegions allow application developers to notify screen readers of a text change in a control that is located apart from the location where the user is working. This is useful, for example, for a <xref:System.Windows.Forms.StatusStrip> control that shows a connection status. If the connection is dropped and the status changes, the developer might want to notify the screen reader.
```

File: docs/framework/whats-new/whats-new-in-accessibility.md [hunk 3]
```diff
@@ -55,9 +65,260 @@ Applications that target versions of the .NET Framework starting with 4.7.1 can
...
+         base.OnTextChanged(e);
+         AutomationNotifications.UiaRaiseLiveRegionChangedEvent(this.AccessibilityObject);
+      }
+   }
+}
+```
+
+**UIA notification events**
+
+The UIA Notification event, introduced in Windows 10 Fall Creators Update, allows your app to raise a UIA event, which leads to Narrator simply making an announcement based on text you supply with the event, without the need to have a corresponding control in the UI. In some scenarios, this is a straightforward way to dramatically improve the accessibility of your app. In can also be useful to notify of the progress of some process that may take a long tinme. For more information about UIA Notification Events, see [Can your desktop app leverage the new UI Notification event?](https://blogs.msdn.microsoft.com/winuiautomation/2017/11/08/can-your-desktop-app-leverage-the-new-uia-notification-event-in-order-to-have-narrator-say-exactly-what-your-customers-need/).
```
Comment (mairaw) [inline docs/framework/whats-new/whats-new-in-accessibility.md]:
```suggestion
The UIA Notification event, introduced in Windows 10 Fall Creators Update, allows your app to raise a UIA event, which leads to Narrator simply making an announcement based on text you supply with the event, without the need to have a corresponding control in the UI. In some scenarios, this is a straightforward way to dramatically improve the accessibility of your app. In can also be useful to notify of the progress of some process that may take a long time. For more information about UIA Notification Events, see [Can your desktop app leverage the new UI Notification event?](https://blogs.msdn.microsoft.com/winuiautomation/2017/11/08/can-your-desktop-app-leverage-the-new-uia-notification-event-in-order-to-have-narrator-say-exactly-what-your-customers-need/).
```

File: docs/framework/whats-new/whats-new-in-accessibility.md [hunk 4]
```diff
@@ -55,9 +65,260 @@ Applications that target versions of the .NET Framework starting with 4.7.1 can
...
+
+The workflow designer includes the following changes in .NET Framework 4.8:
+
+- Users using Narrator will see improvements in FlowSwitch case labels.
+
+- Users using Narrator will see improvements in button descriptions.
+
+- Users who choose High Contrast themes will see improvements in the visibility of the Workflow Designer and its controls, like better contrast ratios between elements and more noticeable selection boxes used for focus elements.
+
+If your application targets .NET Framework 4.7.2 or an earlier version, you can opt into these changes by setting the `Switch.UseLegacyAccessibilityFeatures.3` [AppContext switch](../configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md) to `false` in your application configuration file. For more information, see the [Taking advantage of accessibility enhancements](#taking-advantage-of-accessibility-enhancements) section in this acticle.
```
Comment (mairaw) [inline docs/framework/whats-new/whats-new-in-accessibility.md]:
```suggestion
If your application targets .NET Framework 4.7.2 or an earlier version, you can opt into these changes by setting the `Switch.UseLegacyAccessibilityFeatures.3` [AppContext switch](../configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md) to `false` in your application configuration file. For more information, see the [Taking advantage of accessibility enhancements](#taking-advantage-of-accessibility-enhancements) section in this article.
```

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments.

**Pull request merged**